docs/netobserv_cli.adoc

```diff
@@ -1,9 +1,12 @@
 // Automatically generated by './scripts/generate-doc.sh'. Do not edit, or make the NETOBSERV team aware of the editions.
 :_mod-docs-content-type: REFERENCE
+
 [id="network-observability-cli-usage_{context}"]
```

Comment:
```suggestion

[id="network-observability-netobserv-cli-reference_{context}"]
```

Reply:
This is the unique ID used in the docs, and the one that we have in the links currently so we should keep it consistent to avoid copy/paste issues when there's updates.